packages/bp/src/core/dialog/dialog-engine.ts

```diff
@@ -295,6 +295,11 @@ export class DialogEngine {
     // Check for a timeout node in the current flow
     if (!timeoutNode) {
       timeoutNode = this.findNodeWithoutError(botId, currentFlow, 'timeout')
+      if (!timeoutNode && currentFlow.name.startsWith('skills/') && event.state.context?.previousFlow) {
```

Comment:
@EFF We actually need to recheck because on line 297 we are trying to get the timeoutNode inside of the current flow, skills actually can have a timeoutNode, only if it fails to get something there, we will check if it's a skill, if so, try to get the timeoutNode from the parent flow